packages/blocks/src/api/validation/index.js

```diff
@@ -410,9 +410,17 @@ export const isEqualAttributesOfName = {
 	class: ( actual, expected ) => {
 		// Class matches if members are the same, even if out of order or
 		// superfluous whitespace between.
-		return ! xor(
-			...[ actual, expected ].map( getTextPiecesSplitOnWhitespace )
-		).length;
+		const [ actualPieces, expectedPieces ] = [ actual, expected ].map(
+			getTextPiecesSplitOnWhitespace
+		);
+		const actualDiff = actualPieces.filter(
+			( c ) => ! expectedPieces.includes( c )
+		);
+		const expectedDiff = expectedPieces.filter(
+			( c ) => ! actualPieces.includes( c )
+		);
+
+		return new Set( [ ...actualDiff, ...expectedDiff ] ).size === 0;
```

Comment:
Wouldn't it be better to simply test for `actualDiff.length === 0 && expectedDiff.length === 0`?

Reply:
Good call 👍 Addressed in https://github.com/WordPress/gutenberg/pull/43389/commits/6df8ac5a7f15c943c09614d7a13e1ac0d7471729.